src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -96,6 +96,7 @@ internal static class EnforceOnBuildValues
     public const EnforceOnBuild UseCollectionExpressionForCreate = /*IDE0303*/ EnforceOnBuild.Recommended;
     public const EnforceOnBuild UseCollectionExpressionForBuilder = /*IDE0304*/ EnforceOnBuild.Recommended;
     public const EnforceOnBuild UseCollectionExpressionForFluent = /*IDE0305*/ EnforceOnBuild.Recommended;
+    public const EnforceOnBuild ConvertToInterpolatedString = /*IDE0310*/ EnforceOnBuild.Recommended;
```

Comment:
Can you make this 0320.  i'd like teh whole 0300-19 rance for collection expressions.  thanks!